resources/js/processes/screen-builder/components/form/file-upload.vue

```diff
@@ -93,10 +94,27 @@ export default {
         },
         singleFile: true
       },
+      attrs: {
+        accept: this.type
+      },
       reRenderKey: 0,
     };
   },
   methods: {
+    addFile(file) {
+      if (this.type) {
```

Comment:
Why are we storing the type in a string to begin with? Wouldn't it be cleaner if we store the type as an object from the beginning and just use the `type.includes('ext')` method instead?